agent/tests/conftest.py

```diff
@@ -49,3 +77,11 @@ def mock_sql_session(mock_sql_engine):
         yield session
         # Rollback any changes made in the test
         session.rollback()
+
+
+def pytest_sessionfinish(session, exitstatus):
+    """Clean up temporary migrations directory after all tests."""
+    import shutil
```

Comment:
All imports should be at the top of the file, where possible.

Can you add a `CONTRIBUTING.md` file to the root of the repo and note this down as a style guide rule until we figure out how to lint for it?